docs/python/how-to-send-a-signal-with-start-in-python.md

```diff
@@ -0,0 +1,25 @@
+---
+id: how-to-send-a-signal-with-start-in-python
+title: How to send a Signal-With-Start in Python
+sidebar_label: Signal-With-Start
+description: Signal-With-Start
+tags:
+  - developer-guide
+  - sdk
+  - python
+---
+
+To send a Signal-With-Start in Python, use the [`start_workflow()`](https://python.temporal.io/temporalio.client.client#start_workflow) method and pass the `start_signal` argument with the name of your Signal, instead of a traditional Workflow start.
```

Comment:
```suggestion
To send a Signal-With-Start in Python, use the [`start_workflow()`](https://python.temporal.io/temporalio.client.client#start_workflow) method and pass the `start_signal` argument with the name of your Signal, instead of using a traditional Workflow start.
```